### PR TITLE
[0.12.0] Add Activity Leave

### PR DESCRIPTION
**Pull request by Nubboyy**
image

## Features
- Add Activity Leave
  Resolves FRO-204 [link](https://linear.app/skiso/issue/FRO-204/attendance-add-activity-leave)

### CODE REVIEW

File: components/attendance/AbsenceTypeSelector.tsx
Comment (pixelpxed) [inline components/attendance/AbsenceTypeSelector.tsx:1]: Simple and clean code, never gonna give anyone up on this!

**Review by pixelpxed — APPROVED** [review]
LGTM!

File: public/static/locales/en-US/attendance.json [hunk 1]
```diff
@@ -59,6 +59,7 @@
     "absenceType": {
       "sick": "Sick",
       "covid": "COVID-19",
+      "activity":"Activity",
```
Comment (SiravitPhokeed) [inline public/static/locales/en-US/attendance.json:62]: Add a space between `:` and `"Activity"`. It’s good practice to have Prettier set up on your code editor, and regularly formatting the file with `Shift`+`Alt`+`F`.

Same in the Thai file as well.

**Review by SiravitPhokeed — APPROVED** [review]
I’m honestly not sure about the Present Chip being selected when Activity leave is selected. Like, I get that we should count them as present but I think it’s a little confusing that the Absent Chip opens the Activity Chip but the Activity Chip selects the Present Chip.

Although it is good as is so just format the files and LGTM!

**Pull request merged**